dev/CommonStyles/TestUI/InkToolbarPage.xaml

```diff
@@ -5,7 +5,7 @@
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
     <StackPanel>
-        <InkToolbar TargetInkCanvas="{x:Bind inkCanvas}"/>
+        <InkToolbar TargetInkCanvas="{x:Bind inkCanvas}" Margin="2,0,0,0"/>
```

Comment:
We should add a test to load and show this page as well as click the flyouts to open them, so we ensure this runs in the lab in all OS versions.